imports/ui/components/Message.jsx

```diff
@@ -83,11 +83,34 @@ class Messages extends Component {
 			return (
 				<div className='conversation-container'>
 					<Conversation individualConversations={this.props.messages.activeThread.messagesWithContact} userId={this.props.currentUser._id}/>
-					<input type="text"/>
+					<input type="text" onKeyDown={this.inputKeyPress}/>
```

Comment:
i'll merge this for now but in the future please set the value={this.state.chatMessageInput} and inputkeypress will update the state